standards/javascript.md

```diff
@@ -117,13 +117,13 @@ prefer to use single quotation marks `'`.
 ##### Bad
 
 ```js
-var message = 'oh hai ' + name + "!";
+const message = 'oh hai ' + name + "!";
 ```
 
 ##### Good
 
 ```js
-var message = 'oh hai ' + name + '!';
+const message = 'oh hai ' + name + '!';
```

Comment:
Let's make this a template string `const message = `oh hai ${ name }!`;

Reply:
This is the section about single-quoting strings, though. We should probably have a separate section that addresses template strings.

Reply:
That sounds better, and remove the concatenation here perhaps. It's just about the consistent quoting.

Reply:
I didn't update any of the guide content in this PR but I have another branch that makes exactly this change and expands on the examples a bit.

Reply:
👍